v4/header.html

```diff
@@ -15,7 +15,7 @@ <h1 class="d-md-none">IACR</h1>
   </div>
 </div>
 
-<div id="menubar" class="container-fluid default-menubar">
+<div id="navbar" class="container-fluid">
```

Comment:
navbar seems like a bad choice for this id.

Reply:
Resolved (see commit 401761d1e55008c33404576aeb70b5bd1c07b609)